src/main/java/org/commcare/formplayer/beans/menus/BaseResponseBean.java

```diff
@@ -9,6 +9,8 @@
 
 import org.javarosa.core.services.locale.Localization;
 
+import io.micrometer.core.lang.Nullable;
```

Comment:
should this be `import org.springframework.lang.Nullable;`?

Reply:
good catch, I can't even remember where i got that from.

It actually can't be null. When parsed, if the attribute isn't present, the value is saved as `false`. I removed it in [64d31a8](https://github.com/dimagi/formplayer/pull/1531/commits/64d31a866dd2699c7b8b1ddd00dd2f23f9ed0b59). I also changed it so it's only in the queryResponse since it's not applicable elsewhere.